js/Intents/index.js

```diff
@@ -0,0 +1,27 @@
+import { NativeModules, Platform } from 'react-native';
+
+const { ReactNativePayments } = NativeModules;
+
+const IS_ANDROID = Platform.OS === 'android';
+
+const Intents = {
+	savePaymentMethod(methodData, cardParams) {
+		return new Promise((resolve, reject) => {
+		  if (IS_ANDROID) {
+			return resolve();
+		  }
+	
+		  ReactNativePayments.savePaymentMethod(
+			methodData,
```

Comment:
What do you anticipate will go inside `methodData`?

I'm guessing we are missing the bridge method inside `ReactNativePayments.m`

Reply:
gateway config and client_secret